akka-remote/src/main/resources/reference.conf

```diff
@@ -147,41 +180,41 @@ akka {
         compression {
           # possibility to disable compression by setting this to off
           enabled = on
-          
+
           # unlocks additional very verbose debug logging of compression events (on DEBUG log level)
           debug = off
```

Comment:
do we need this? can it be a compile time constant, if it's only for our own debugging purposes?